README.md

```diff
@@ -0,0 +1,217 @@
+# Datadog Agent Integrations
+
+Collecting data is cheap; not having it when you need it can be very expensive. So we recommend instrumenting as much of your systems and applications as possible. This integrations repository will help you do that by making it easier to create and share new integrations for [Datadog](https://www.datadoghq.com).
+
+# Adding New Integrations
+
+## Requirements
+
+You will need a working [Ruby](https://www.ruby-lang.org) environment. For more information on installing Ruby, please reference [the Ruby installation documentation](https://www.ruby-lang.org/en/documentation/installation/).
+
+You will also need [Wget](https://www.gnu.org/software/wget/). Wget is already installed on most Linux systems and is easy to install on Mac using [Homebrew](http://brew.sh/) or on Windows using [Chocolatey](https://chocolatey.org/).
+
+## Setup
+
+We've written [a gem](https://rubygems.org/gems/datadog-sdk-testing) and a set of scripts to help you get set up, ease development, and provide testing. To begin:
+
+1. Clone the Github repository
+2. Run `gem install bundler`
+3. Run `bundle install`
+
+Once the required Ruby gems have been installed by Bundler, you can easily create a [Python](https://www.python.org/) environment:
+
+1. Run `rake setup_env`. This will install a Python virtual environment along with all the components necessary for integration development. This will also create an entry for your new integration in our `.travis.yml` and `circle.yml` continuous integration files to ensure that your tests are run whenever new builds are created.
```

Comment:
`rake setup_env` will only install the development environment.

It's `rake generate:skeleton[integration_name]` that will generate an entry for a new integration.